src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Protocol/DelegatedHoverParams.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
+
+internal record DelegatedHoverParams(VersionedTextDocumentIdentifier HostDocument, Position ProjectedPosition, RazorLanguageKind ProjectedKind);
```

Comment:
lol, the file scoped namespace makes this file look kinda silly 😛

Maybe we should combine this and DelegatedRenameParams into a single file called DelegatedParams.cs or something?

Reply:
I'm not opposed to that for record/POCO types

Reply:
Done